app/models/message.rb

```diff
@@ -9,6 +9,22 @@ class Message < ActiveRecord::Base
     :contents,
     presence: true
 
+  def with_delegates(*delegates)
+    delegates = delegates.first if delegates.first.class == Array
+
+    delegates.each do |delegate|
+      self.define_singleton_method("include_#{delegate}") do
```

Comment:
This is probably pretty weird… it doesn't persist? We need data to persist or it will never make it to delayed_job.